pom.xml

```diff
@@ -69,5 +109,4 @@
         </profile>
     </profiles>
 
-
 </project>
```

Comment:
Add a new blank line at the end of the file

Reply:
added